daily-emailing/html_content.py

```diff
@@ -53,6 +54,12 @@ def generate_html_with_links() -> str:
     return html
 
 
+def add_unsubscribe_link() -> str:
```

Comment:
Could you also add a test for this new function

Reply:
I'll do that now!!